src/main/java/game/Player.java

```diff
@@ -0,0 +1,89 @@
+package game;
+
+import game.stores.ItemStore;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class Player implements Interactable {
+
+    private String name;
+    private List<String> inventory;
+    private PlayerStats playerStats;
+
+    public Player() {
+        inventory = new ArrayList<>();
+        playerStats = new PlayerStats();
+        playerStats.setMaxHealthPoints(100);
+        playerStats.setMinHealthPoints(0);
+        playerStats.setHealthPoints(100);
+    }
+
+    public Player(String name) {
+        this();
+        this.name = name;
+    }
+
+    /*
+    User Interaction Methods
+     */
+
+    @Override
+    public boolean onCommand(Command command, Callback callback) {
+        if (command.getAction().equals("inventory")) {
+
+            StringBuilder inventoryString = new StringBuilder("Inventory:\n=============\n");
+
+            for (String itemName : inventory) {
+                inventoryString.append("\t").append(itemName).append("\n");
+            }
+
+            inventoryString.append("================");
+
+            callback.onMessage(inventoryString.toString());
+
+            return true;
+        } else if (command.getAction().equals("stats")) {
+
+            callback.onMessage(playerStats.toString());
+            return true;
+        }
+
+        return false;
+    }
+
+    @Override
+    public List<String> listCommands(Game game, List<String> addToThisList) {
```

Comment:
You have used the "addToThisList" variable multiple times and thus it might be a good idea to rename it to be more descriptive of its purpose.

Reply:
I will be removing `addToThisList`